config/initializers/rack_attack.rb

```diff
@@ -82,8 +82,12 @@ def paging_request?
     req.authenticated_user_id if req.post? && req.path.match?(%r{\A/api/v\d+/media\z}i)
   end
 
-  throttle('throttle_media_proxy', limit: 30, period: 10.minutes) do |req|
-    req.throttleable_remote_ip if req.path.start_with?('/media_proxy')
+  throttle('throttle_authenticated_media_proxy', limit: 200, period: 10.minutes) do |req|
+    req.authenticated_user_id if req.path.start_with?('/media_proxy')
```

Comment:
It would be useful to add some specs to the rack attack spec for these scenarios.

Separately - should this line have a check for authenticated?

Reply:
Right ... so doesn't it need to check for that?

I'd say start by adding some specs showing the changed behavior we want, and then sort out these methods after.